settings.gradle.kts

```diff
@@ -1,14 +1,28 @@
 pluginManagement {
+    val agp = "7.0.0-alpha09"
+    val kotlin = "1.4.20"
+
     plugins {
+        id("com.android.application") version agp
         id("org.jmailen.kotlinter") version "3.3.0"
         id("com.squareup.sqldelight") version "1.4.4"
+        id("com.prof18.kmp.fatframework.cocoa") version "0.0.1"
+        kotlin("multiplatform") version kotlin
+        kotlin("native.cocoapods") version kotlin
+        kotlin("android") version kotlin
```

Comment:
I moved declaring Gradle plugins here from `buildscript` as:
1. `buildscript` is a legacy method
2. Having some of plugins declared in `buildscript` and some in `pluginManagement` block resulted with unpredicted behaviour (like lowering Kotlin plugin to `1.4.0`)